core.multiview/src/org/netbeans/core/multiview/SplitAction.java

```diff
@@ -81,7 +80,7 @@ public JMenuItem getMenuPresenter() {
     public JMenuItem getPopupPresenter() {
 	return getSplitMenuItem();
     }
-    
+
```

Comment:
... these whitespaces don't need to be changed.

Reply:
Can't see any whitespace here :/